pkg/auth/userregistry/identitymapper/strategy_claim.go

```diff
@@ -0,0 +1,66 @@
+package identitymapper
+
+import (
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kerrs "k8s.io/kubernetes/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/util/sets"
+
+	"github.com/openshift/origin/pkg/user"
+	userapi "github.com/openshift/origin/pkg/user/api"
+	userregistry "github.com/openshift/origin/pkg/user/registry/user"
+)
+
+type StrategyClaim struct {
```

Comment:
GoDoc here might be helpful, before reading further I thought this would be stomp strategy.